server/models/userModel.js

```diff
@@ -7,7 +7,7 @@ const userSchema = new Schema(
   {
     firstName: { type: String, unique: false },
     lastName: { type: String, unique: false },
-    password: { type: String, unique: false, minlength: 7 },
```

Comment:
Why we're getting rid of minLength here?

Reply:
Yes, I got rid of that because when the form goes on validation after clicking the submit button, the error pops up in the server-side first which is in console, and front end validation error doesn't show up.